docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -0,0 +1,64 @@
+<button uxFocusIndicator
+        tabindex="0"
+        class="icon-demo btn btn-link button-secondary btn-text"
+        [uxPopover]="iconCodeSnippet"
+        popoverClass="icon-code-snippet-popover"
+        placement="bottom">
+    <i class="{{ iconClass }} p-r-xs" [ngClass]="classname"></i>
+    <span>{{ iconset === 'ux-icons' ? name : classname }}</span>
+</button>
+
+<ng-template #iconCodeSnippet>
+    <table class="icon-code-snippets-table">
+        <tbody>
+            <tr *ngIf="iconset === 'ux-icons'">
+                <td class="p-l-xs">
+                    <code>
+                        &lt;ux-icon name="{{name}}"&gt;&lt;/ux-icon&gt;
+                    </code>&nbsp;&nbsp;
+                </td>
+                <td>
+                    <i class="ux-icon ux-icon-clipboard m-r-xs"
+                       [uxTooltip]="tooltipTemplate"
+                       (click)="copy('ux-icon-component')">
+                    </i>
+                </td>
+            </tr>
+            <tr *ngIf="iconset === 'ux-icons'">
+                <td class="p-l-xs">
+                    <code>
+                        &lt;i class="ux-icon {{classname}}"&gt;&lt;/i&gt;
+                    </code>&nbsp;&nbsp;
+                </td>
+                <td>
+                    <i class="ux-icon ux-icon-clipboard m-r-xs"
+                       [uxTooltip]="tooltipTemplate"
+                       (click)="copy('ux-icon')">
+
+                    </i>
+                </td>
+            </tr>
+            <tr *ngIf="iconset !== 'ux-icons'">
```

Comment:
Condition should be `iconset === 'hpe-icons'` since you have explicitly cited `hpe-icon` below.

Reply:
updated to above